src/System.Numerics.Vectors/src/Configurations.props

```diff
@@ -1,8 +1,8 @@
-﻿<Project>
+﻿<Project DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp-Unix;
       netcoreapp-Windows_NT;
+      netcoreapp-Unix;
```

Comment:
We no longer need to cross compile per OSGroup, could you change this to just one `netcoreapp` ridless entry?

Reply:
I tried that and compilation failed with:
> C:\Users\tagoo\.nuget\packages\microsoft.dotnet.build.tasks.configuration\5.0.0-beta.19523.3\build\Microsoft.DotNet.Build.Tasks.Configuration.targets(201,5): error : Could not find a configuration for ProjectReference 'C:\repos\corefx\eng\restore\runtime\runtime.depproj' from configurations 'netcoreapp3.0-Windows_NT;netcoreapp3.0-Unix;netcoreapp-Windows_NT;netcoreapp-Unix;mono' when building 'System.Numerics.Vectors' for configuration 'netcoreapp-Debug' . [C:\repos\corefx\src\System.Numerics.Vectors\src\System.Numerics.Vectors.csproj]

Reply:
A right, cause we transform the `ReferenceFromRuntime Include="System.Private.CoreLib"` to a project reference to `runtime.depproj` and that is RID specific as it restores the necessary corelib. Disregard then.

Reply:
If only we had a System.Private.Corelib contract... 😉